packages/client/src/admin/Apps/Apps.tsx

```diff
@@ -0,0 +1,265 @@
+import * as React from 'react';
+import { get } from 'lodash';
+import {
+  Icon,
+  Layout,
+  Card,
+  Row,
+  Col,
+  Button,
+  Input,
+  Typography,
+  Tag,
+  Divider,
+  notification,
+} from 'antd';
+import { graphql } from 'react-apollo';
+import { compose } from 'recompose';
+
+import { errorHandler } from 'utils/errorHandler';
+import Breadcrumbs from 'components/share/breadcrumb';
+import PhAppTemplate from 'interfaces/phAppTemplate';
+import { ActionBtn, ClearBoth, Left } from 'components/apps/detail';
+import { GetPhAppTemplates } from 'queries/PhAppTemplate.graphql';
+import AppLogo from 'components/apps/appLogo';
+
+import { ImportPhAppTemplateFromURL } from './apps.graphql';
+
+const { Search } = Input;
+const { Text } = Typography;
+
+const GITHUB_REGEX =
+  /^https:\/\/github.com\/([A-Za-z0-9._-]+\/[A-Za-z0-9._-]+)\/blob\/(.+)/;
```

Comment:
I think here might have an edge case: the user pastes a URL like `http://github.com` will encounter any error?